controllers/flinkcluster_converter.go

```diff
@@ -753,6 +755,52 @@ func convertFromSavepoint(jobSpec *v1beta1.JobSpec, clusterStatus *v1beta1.Flink
 	return jobSpec.FromSavepoint
 }
 
+func convertJobManagerInitContainers(jobManagerSpec *v1beta1.JobManagerSpec) []corev1.Container {
+	var initContainers = []corev1.Container{}
+	// Add jobManagerSpec level volume mounts to each init container if there is no
+	// conflict.
+	for _, initContainer := range jobManagerSpec.InitContainers {
```

Comment:
Can we extract this logic into a helper method which can be shared by all 3 methods?

Reply:
This is done.